src/app/space/analyze/analyze-overview/analyze-overview.component.html

```diff
@@ -5,7 +5,7 @@
         <fabric8-edit-space-description-widget [userOwnsSpace]='userOwnsSpace()'></fabric8-edit-space-description-widget>
       </div>
       <div class="col-sm-2">
-        <div *ngIf="userOwnsSpace()">
+        <div *ngIf="userOwnsSpaceBool">
```

Comment:
See comments below about using a `get` method for the `userOwnsSpace` boolean, and then this could read something like: `*ngIf="userOwnsSpace"` etc.